### PR TITLE
Editor Panel: Change - Make search input sticky (ED-1677)

### PR DESCRIPTION
**Pull request by StyleShit**
Before:

image

After:

image

### CODE REVIEW

File: assets/dev/scss/direction/editor-dark-mode.scss [hunk 1]
```diff
@@ -276,6 +276,10 @@ select {
 		}
 	}
 
+	#elementor-panel-elements-search-area {
```
Comment (matipojo) [inline assets/dev/scss/direction/editor-dark-mode.scss:279]: Nice! Please add screenshots before/after.

(Like here: https://github.com/elementor/elementor/pull/13586)

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/scss/direction/editor-dark-mode.scss (hunk 1 above)
Comment (StyleShit) [reply assets/dev/scss/direction/editor-dark-mode.scss:279]: Before:

image

After:

image

Comment (StyleShit) [reply assets/dev/scss/direction/editor-dark-mode.scss:279]: Done

File: assets/dev/scss/editor/panel/_elements.scss [hunk 2]
```diff
@@ -40,9 +40,16 @@
 	}
 }
 
+#elementor-panel-elements-search-area {
+	position: sticky;
+	top: 0;
+	padding: 10px;
+	background-color: $editor-background;
+	z-index: 10;
```
Comment (matipojo) [inline assets/dev/scss/editor/panel/_elements.scss:48]: Use `$first-layer`

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/panel/_elements.scss (hunk 2 above)
Comment (StyleShit) [reply assets/dev/scss/editor/panel/_elements.scss]: Done

**Review by matipojo — DISMISSED** [review]

**Review by kobizz — APPROVED** [review]

**Review by kobizz — DISMISSED** [review]

**Review by kobizz — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**